### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by jahagirdar** [cocotb/cocotb-bus#67] [General]
- [ ] Update Documentation
- [ ] Fix all references to deprecated method's in cocotb
Make this repo the base class for all VIP Development effort.
- [ ] Get Inputs from Existing users on pain points. And add items to this roadmap.
- [ ] Get inputs from VIP developers who do not use these classes on the reason for not using it and add pain points to roadmap.
- [ ] Add complete RAL model and test generation framework (based on peakrdl?) Or should that be a different repo?

**Comment by cmarqu** [comment]
Once compatible again, run add cocotb-bus in `cocotb/.github/workflows/ecosystem-compat.yml`.
